### PR TITLE
add new bundle ecoco/sylius-base-price-plugin

### PR DESCRIPTION
**Pull request by andrejsstepanovs**
| Q             | A
| ------------- | ---
| License       | MIT

If I do proceed to fix Travis CI build then I will just break my recipe. Because I will need to enable and configure so many unrelated bundles. This will definitely be destructing for many sylius shops that oped to not use them.

I tried recipe using given instructions and it works exactly as expected.
Not sure what I can do to get this thing merged automatically. Any pointers are welcomed.

### CODE REVIEW

**Comment by andrejsstepanovs** [comment]
If I do proceed to fix Travis CI build then I will just break my recipe. Because I will need to enable and configure so many unrelated bundles. This will definitely be destructing for many sylius shops that oped to not use them.

I tried recipe using [given instructions](1) and it works exactly as expected.
Not sure what I can do to get this thing merged automatically. Any pointers are welcomed.

[1]: https://flex.symfony.com/r/github.com/symfony/recipes-contrib/1146

**Comment by andrejsstepanovs** [comment]
Can this be checked and accepted by human being please?

**Comment by Nyholm** [comment]
The CI jobs are not really happy. 

Could you rebase this PR to make sure the updated CI jobs are green?
I need it to work on Symfony 4 or 5.

**Comment by Nyholm** [comment]
Hey. Dont forget about this PR.

File: ecoco/sylius-base-price-plugin/1.2/manifest.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "bundles": {
+        "Ecocode\\SyliusBasePricePlugin\\EcocodeSyliusBasePricePlugin": ["all"]
+    },
+    "copy-from-package": {
```
Comment (vvasiloi) [inline ecoco/sylius-base-price-plugin/1.2/manifest.json:5]: Please add the files to the recipe and use `copy-from-recipe` instead.

**Review by vvasiloi — CHANGES_REQUESTED** [review]
Hey @andrejsstepanovs, if you still want this merged, please rebase the branch and make the requested changes.

**Comment by fabpot** [comment]
Closing as there is no more activity and the PR cannot be merged as is. Feel free to reopen if you like.

**Pull request closed without merging**